### PR TITLE
The ultimate emote sound

### PR DESCRIPTION
**Pull request by Sebbe9123**
Long been requested, your dreams fulfilled.
*Moan now has a sound attached to it.

### CODE REVIEW

**Comment by SpadesNeil** [comment]
p l e a s e  no

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -432,6 +432,9 @@
 			else
 				message = "moans!"
 				m_type = 2
+				//vorestation edit start
+				playsound(loc, 'sound/voice/moan.ogg', 50, 1, -1)
+				//vorestation edit stop
```
Comment (Novacat) [inline code/modules/mob/living/carbon/human/emote.dm]: No need to do start/stop on a single line vorestation edit.

Comment (Sebbe9123) [reply code/modules/mob/living/carbon/human/emote.dm]: D-does it matter?

Comment (Novacat) [reply code/modules/mob/living/carbon/human/emote.dm]: I prefer for edit markings to be consistent as possible.

Comment (Sebbe9123) [reply code/modules/mob/living/carbon/human/emote.dm]: done

**Comment by SpadesNeil** [comment]
This is a little too cringy, and unlike the gnomes, you don't have to find this first to spam the hell out of it.

Gonna reject this.

**Pull request closed without merging**